packages/sui-widget-embedder/downloader/index.js

```diff
@@ -107,24 +107,38 @@
     })
   }
 
-  function matchPathnameWithRegExp(regExp) {
-    var pathname = window.location.pathname
+  function matchNameWithRegExp(name, regExp) {
+    if(!regExp) {
```

Comment:
I think the linter should warn you about this line to be `if (!regExp) {` instead `if(!regExp) {`

Reply:
Fixed